regression-tests/test-results/msvc-2022-c++20/pure2-assert-expected-not-null.cpp.output

```diff
@@ -1,20 +1,22 @@
 pure2-assert-expected-not-null.cpp
 pure2-assert-expected-not-null.cpp2(7): error C2039: 'expected': is not a member of 'std'
-predefined C++ types (compiler internal)(348): note: see declaration of 'std'
+predefined C++ types (compiler internal)(347): note: see declaration of 'std'
 pure2-assert-expected-not-null.cpp2(7): error C2062: type 'int' unexpected
 pure2-assert-expected-not-null.cpp2(7): error C2143: syntax error: missing ';' before '{'
 pure2-assert-expected-not-null.cpp2(7): error C2143: syntax error: missing ';' before '}'
 pure2-assert-expected-not-null.cpp2(9): error C2065: 'ex': undeclared identifier
 pure2-assert-expected-not-null.cpp2(9): error C2672: 'cpp2::impl::assert_not_null': no matching overloaded function found
-..\..\..\include\cpp2util.h(632): note: could be 'decltype(auto) cpp2::impl::assert_not_null(_T0 &&)'
+..\..\..\include\cpp2util.h(638): note: could be 'decltype(auto) cpp2::impl::assert_not_null(_T0 &&)'
 pure2-assert-expected-not-null.cpp2(14): error C2039: 'expected': is not a member of 'std'
-predefined C++ types (compiler internal)(348): note: see declaration of 'std'
+predefined C++ types (compiler internal)(347): note: see declaration of 'std'
 pure2-assert-expected-not-null.cpp2(14): error C2062: type 'int' unexpected
 pure2-assert-expected-not-null.cpp2(14): error C2143: syntax error: missing ';' before '{'
 pure2-assert-expected-not-null.cpp2(14): error C2039: 'unexpected': is not a member of 'std'
-predefined C++ types (compiler internal)(348): note: see declaration of 'std'
-pure2-assert-expected-not-null.cpp2(14): error C3861: 'unexpected': identifier not found
+predefined C++ types (compiler internal)(347): note: see declaration of 'std'
+pure2-assert-expected-not-null.cpp2(14): error C2660: 'unexpected': function does not take 1 arguments
+C:\Program Files\Microsoft Visual Studio�2nterprise\VC\Tools\MSVC.40.33807\includeh.h(33): note: see declaration of 'unexpected'
```

Comment:
[Explanation] This path was always indicated as different despite the test file being updated. My conjecture is that this is due to the strange characters in the path.